net-irc/bitchx/bitchx-1.2.1.recipe

```diff
@@ -0,0 +1,75 @@
+SUMMARY="A text-based IRC client"
+DESCRIPTION="bitchx is an IRC client written in the C programming language.\
+It uses a text-mode user interface.
+Features:
+* Built-in ANSI color.
+* Ease of use. BitchX adds dozens of useful command aliases to reduce typing.
+* Built-in notify, protection, bot, and shit lists.
+* Built-in mass commands and tools.
+* Extended set of DCC commands and built-in CDCC offering.
+* Built-in link looker.
+* Extended scripting functionality, including functions found nowhere else.
+* Huge installed user base, an abundance of scripts, and a dedicated help \
+channel (#BitchX on EFNET).
+* Built-in screen client, allowing you to detach and re-attach to an irc session."
+HOMEPAGE="http://www.bitchx.com/"
+COPYRIGHT="1990 Michael Sandroff
+	1991, 1992 Troy Rollo
+	1992-1996 Matthew Green
+	1993, 1997 Jeremy Nelson
+	1994 Jake Khuon
+	1995, 1998 Jeremy Nelson and others (\"EPIC Software Labs\")
+	1996-2004 Colten Edwards et al.
+	2008-2014 Kevin Easton et al."
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="http://www.bitchx.com/download/bitchx-$portVersion.tar.gz"
+CHECKSUM_SHA256="2d270500dd42b5e2b191980d584f6587ca8a0dbda26b35ce7fadb519f53c83e2"
+PATCHES="bitchx-$portVersion.patchset"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
```

Comment:
Suggestion:
```diff
-ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+ARCHITECTURES="x86_gcc2 ?x86 x86_64"
```
Good news! A tiny patch (swapping lines 999 and 1000 of `bitchx-1.2.1/source/istatus.c`) is enough to make it build with gcc2:
```diff
diff --git a/source/status.c b/source/status.c
index 36bba31..20bccf7 100644
--- a/source/status.c
+++ b/source/status.c
@@ -996,8 +996,8 @@ static      char    *status_notify_windows(Window *window)
 
        if (get_int_var(SHOW_STATUS_ALL_VAR) || window == window->screen->current_window)
        {
-               *notes = 0;
                Window *notify_win = NULL;
+               *notes = 0;
                while ((traverse_all_windows(&notify_win)))
                {
                        if (notify_win->miscflags & WINDOW_NOTIFIED)
```